internal/storage/v2/grpc/dependency_storage.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package jaeger.storage.v2;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "model.proto";
+
+option go_package = "storage";
+
+message GetDependenciesRequest {
+  // start_time is the start of the time interval to search for the dependencies.
+  google.protobuf.Timestamp start_time = 1 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  // end_time is the end of the time interval to search for the dependencies.
+  google.protobuf.Timestamp end_time = 2 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+}
+
+message GetDependenciesResponse {
+  repeated jaeger.api_v2.DependencyLink dependencies = 1 [(gogoproto.nullable) = false];
```

Comment:
I don't think we should create cross-dependencies. It's a simple struct, let's just define it here.